src/configuration.js

```diff
@@ -15,6 +15,8 @@ const {
   REACT_APP_NATIVE_TOKEN_NAME,
   REACT_APP_DONATION_COLLECT_COUNT_LIMIT = 10,
   REACT_APP_DONATION_DELEGATE_COUNT_LIMIT = 10,
+  CONVERSATION_MESSAGE_SIZE_MIN_LIMIT = 0,
```

Comment:
As min length is fixed in feathers, please remove CONVERSATION_MESSAGE_SIZE_MIN_LIMIT